bin/test-php-watch.sh

```diff
@@ -13,5 +13,5 @@ while true; do
 	echo "Running phpunit tests for $(tput bold)$plugin_slug$(tput sgr0):"
 	# TODO: Interrupt when a change is made while running tests or re-run if change made since tests started running.
 	# Note: This is calling phpunit directly and not the composer script due to extra noise it outputs.
-	npm run wp-env --silent -- run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance -- vendor/bin/phpunit --testsuite "$plugin_slug" "$@"
+	npm run wp-env --silent -- run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance -- vendor/bin/phpunit --testsuite "$plugin_slug" --verbose "$@"
```

Comment:
Why is this changed here? Mostly asking since it'll affect all plugins.

Reply:
I added it because without `--verbose` the messages communicated by `markTestIncomplete()` are not printed:

https://github.com/WordPress/performance/blob/db322a37e675aeea54db7afba8a31ac0c40e6393/plugins/optimization-detective/tests/class-optimization-detective-test-helpers.php#L318-L319

However, I just realized this can still be passed as extra args:

```
npm run test-php-watch  -- --verbose
```

So I've reverted this.